assets/airports/saez.json

```diff
@@ -865,207 +987,4568 @@
+                ["ual/long", 3]
             ]
         },
         {
             "origin": "",
             "destination": "saez",
             "category": "arrival",
-            "route": "ESLAN..PTA..EZE",
-            "altitude": 12000,
+            "route": "DUR..PONPA..PAPIX.PAPIX.SAEZ",
+            "altitude": 24000,
             "speed": 250,
-            "method": "cyclic",
-            "rate": 3,
-            "offset": 0,
+            "method": "random",
+            "rate": 2,
             "airlines":  [
-                ["arg", 10],
-                ["lan", 5],
-                ["dsm", 5],
-                ["tam", 3],
-                ["afr/long", 1],
-                ["klm/long", 1]
+                ["aea/long", 3],
+                ["afr/long", 4],
+                ["arg", 40],
+                ["aut", 29],
+                ["aza/long", 3],
+                ["azu", 2],
+                ["baw/long", 3],
+                ["dlh/long", 2],
+                ["dsm", 7],
+                ["glo", 20],
+                ["ibe/long", 6],
+                ["klm/long", 3],
+                ["qtr/long", 3],
+                ["tam", 9],
+                ["thy/long", 3],
+                ["uae", 3]
             ]
         },
         {
             "origin": "",
             "destination": "saez",
             "category": "arrival",
-            "route": "_PAPIX056034..PAPIX..EZE",
-            "altitude": 10000,
+            "route": "ROS..MULTA..SNT.SNT.SAEZ",
+            "altitude": 24000,
             "speed": 250,
-            "method": "cyclic",
-            "rate": 7.5,
-            "offset": 50,
+            "method": "random",
+            "rate": 2,
             "airlines":  [
-                ["arg", 10],
-                ["tam", 8],
-                ["dsm", 5],
-                ["afr/long", 3],
-                ["klm/long", 1],
-                ["ibe/long", 3],
-                ["dlh/long", 2],
-                ["baw/long", 4],
-                ["aza/long", 3],
-                ["uae", 1]
+                ["aal/long", 12],
+                ["aca/long", 2],
+                ["amx/long", 3],
+                ["arg", 40],
+                ["aut", 29],
+                ["ava/long", 3],
+                ["bov", 2],
+                ["cmp", 8],
+                ["dal/long", 3],
+                ["dsm", 7],
+                ["lan", 3],
+                ["lne", 2],
+                ["lpe", 4],
+                ["ual/long", 3]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "saez",
+            "category": "arrival",
+            "route": "PULAS..EGANI..VALOS.VALOS.SAEZ",
+            "altitude": 24000,
+            "speed": 250,
+            "method": "random",
+            "rate": 1,
+            "airlines":  [
+                ["arg", 40],
+                ["aut", 29],
+                ["dsm", 7]
             ]
         }
-    ]
+    ],
+    "maps": {
+        "base": [
```

Comment:
> Debo anadir que (si es posible), siempre tenemos mostrar lineas extendidas para cada pista que alguna vez aceptaria (arrivals).

It is possible but I honestly can't be bothered 😅  they take really long to make and I don't really think they're necessary, mainly because those runways don't have an ILS system and are barely ever used.